### ISSUE TITLE
hlsclt status - error

### ISSUE THREAD

**Issue by vmayoral** [benjmarshall/hlsclt#9]
```bash
$ hlsclt status
Traceback (most recent call last):
  File "/usr/local/bin/hlsclt", line 11, in <module>
    load_entry_point('hlsclt==1.0.1', 'console_scripts', 'hlsclt')()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/hlsclt/report_commands/report_commands.py", line 141, in status
    print_project_status(ctx)
  File "/usr/local/lib/python2.7/dist-packages/hlsclt/report_commands/report_commands.py", line 96, in print_project_status
    project_status = gather_project_status(ctx)
  File "/usr/local/lib/python2.7/dist-packages/hlsclt/report_commands/report_commands.py", line 70, in gather_project_status
    with click.open_file(config["project_name"] + "/solution" + str(solution_num) + "/sim/report/" + config["top_level_function_name"] + "_cosim.rpt","r") as f:
  File "/usr/local/lib/python2.7/dist-packages/click/utils.py", line 324, in open_file
    atomic=atomic)
  File "/usr/local/lib/python2.7/dist-packages/click/_compat.py", line 487, in open_stream
    return open(filename, mode), True
IOError: [Errno 2] No such file or directory: 'proj_simple_adder_pynq-z2/solution1/sim/report/simple_adder_cosim.rpt'
```

**Comment by vmayoral** [comment]
Addressed through https://github.com/vmayoral/hlsclt/commit/a03c13d250211b80e3b45fffa8922bc09672a56e

**Comment by vmayoral** [comment]
https://github.com/benjmarshall/hlsclt/pull/10

**Comment by benjmarshall** [comment]
Reopening this issue, as we have more info now. Appears to be a be a bug specific to python 2 where an IOError is raised as opposed to the OSError in python 3. Will be fixed by #10 .

**Comment by benjmarshall** [comment]
Fixed by #10 which has now been merged into develop pending release.
